parachains/runtimes/collectives/collectives-polkadot/src/constants.rs

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2021 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod account {
+	use frame_support::PalletId;
+	use sp_runtime::AccountId32;
+
+	/// Relay Chain treasury pallet id, used to convert into AccountId
+	pub const RELAY_TREASURY_PALL_ID: PalletId = PalletId(*b"py/trsry");
+	/// account used to temporarily deposit slashed imbalance before teleporting
+	pub const SLASHED_IMBALANCE_ACC_ID: AccountId32 = AccountId32::new([7u8; 32]);
+}
+
+pub mod currency {
+	use polkadot_core_primitives::Balance;
+	use polkadot_runtime_constants as constants;
+
+	/// The existential deposit. Set to 1/10 of its parent Relay Chain (v9200).
```

Comment:
```suggestion
	/// The existential deposit. Set to 1/10 of its parent Relay Chain.
```

Not sure if the version meant anything here?

Reply:
Just to reference when this value was taken.

Reply:
Ah, I see what you mean :facepalm: . I wrote this comment in Statemint before `polkadot_runtime_constants` existed and we had just hardcoded it in. And it just survived the addition of this crate.